test/PublishCommand.js

```diff
@@ -204,6 +204,85 @@ describe("PublishCommand", () => {
         }
       }));
     });
+
+    it("should use semver increments when passed to repoVersion flag", (done) => {
```

Comment:
Just a suggestion, but feel it would be worth refactoring these tests to smaller assertion blocks / calls rather than the larger fixture > result chunks. CLI tools with lots of arg permutations is hard to test all paths